### PR TITLE
[SITES-567] Implement content links using liquid templates

### PR DESCRIPTION
**Pull request by tessereth**
Entering the content body
```
{% assign careers = nodes['a183986f-5191-46d4-8cc8-d8bf30ab055b'] %}
{% assign people = nodes['eca9b7c8-6774-4ab3-954d-be3da0c654ed'] %}

## The following pages are in this section

### {{ careers.link_to }}

{{ careers.short_summary }}

### {{ people.link_to }}

{{ people.short_summary }}
```
Results in a published page of
image

Entering invalid content results in an error being displayed on the submission page and the error being highlighted in red.
```
Link 1: {{ nodes['bad'].link_to }}  
Link 2: {{ nodes['bad2'].link_to }}
```
image

Note: As it currently stands, getting the UUID of a node is rather a pain. We should probably have a widget on the submission form that lets you search by name/path. But this PR is big enough as it is.

### CODE REVIEW

File: EDITOR.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+See https://shopify.github.io/liquid/ for a basic introduction to Liquid.
+
+### Links
+
+To create a link to a page, you need to find the id of the target page.
+This can be found on the editorial view of the page.
+
+To link to a page, using the title of the page as the link text:
+
+    {{ nodes['123e4567-e89b-12d3-a456-426655440000'.link_to }}
```
Comment (ltankey) [inline EDITOR.md]: Missing square bracket.

**Comment by tessereth** [comment]
Update changelog post 1.3.0 release

**Comment by freshtonic** [comment]
This looks great @tessereth. LGTM. 

So technically this was a spike but it looks like it's well tested and as production-ready as anything else. Do we merge it?

**Comment by ltankey** [comment]
LGTM - it works, so I think we roll it in. We probably need input from content folks but it's something we can put on their radar without holding up a deploy.

**Pull request merged**